src/main/resources/templates/challenge.html

```diff
@@ -15,12 +15,185 @@
 		<a href="/dashboard" class="primary"><i class="fa fa-graduation-cap"></i>Submit</a>
 	</div>
 </div>
-<div class="challenge-tri">
-<div><textarea class="challenge-input" id="challenge-input">
-// Challenge: Type something that triggers the pretty print.
-// Try something like "var a = document.getElementById('hi');"
-</textarea></div>
-	<div><pre><code id="challenge-pretty" class="js"></code></pre></div>
+<div class="challenge-dual-pane">
+	<div class="left">
+		<div class="challenge-side-header">
+			<div class="title">Simple Dependency Injection</div>
+			<div class="subtitle">Java Challenge</div>
+		</div>
+		<div class="challenge-side-body">
+			<div class="block">
+				Implement a simple dependency injection library that has a single
+				<code>#provide</code> method. You may assume the following:
+			</div>
+			<ul class="block">
+				<li>All injectable constructors are annotated with <code>@Inject</code>.</li>
+				<li>A class named <code>InjectException</code> exists and extends <code>Exception</code>.</li>
+				<li>Only the core Java JDK is allowed. All versions are allowed.</li>
+			</ul>
+			<div class="block">
+				Any other non-listed assumptions must be handled appropriately.
+			</div>
+		</div>
+	</div>
+	<div class="right">
+		<div>
+			<textarea class="code-highlight" id="challenge-input">import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
```

Comment:
Cool! Seemed to fix most of the problems. There are still some red underlining, but here's an interesting one -- 
<img width="1092" alt="screen shot 2017-05-25 at 1 58 56 am" src="https://cloud.githubusercontent.com/assets/13812345/26443557/c6bc6eb0-40ed-11e7-97bb-2298c6e62415.png">

So there's the red underline, but if I try deleting one character, this is what happens --
<img width="1076" alt="screen shot 2017-05-25 at 1 59 53 am" src="https://cloud.githubusercontent.com/assets/13812345/26443579/e201f4ba-40ed-11e7-8498-0e3543aa7b2d.png">